dashboard/src/main/java/com/google/cloud/tools/opensource/dashboard/DashboardMain.java

```diff
@@ -82,21 +86,41 @@
   public static final String TEST_NAME_GLOBAL_UPPER_BOUND = "Global Upper Bounds";
   public static final String TEST_NAME_DEPENDENCY_CONVERGENCE = "Dependency Convergence";
 
-  public static void main(String[] args)
-      throws IOException, TemplateException, RepositoryException, URISyntaxException {
-
-    Path output = generate();
+  /**
+   * Generates a code hygiene dashboard for a BOM. This tool takes a path to pom.xml of the BOM as
+   * an argument or Maven coordinates to a BOM.
+   */
+  public static void main(String[] arguments)
+      throws IOException, TemplateException, RepositoryException, URISyntaxException,
+          PlexusContainerException, ComponentLookupException, ProjectBuildingException {
+    if (arguments.length != 1) {
+      System.err.println("Please specify path to pom.xml or Maven coordinates for a BOM.");
+      return;
+    }
+    String bomInput = arguments[0];
+    Path output = bomInput.endsWith("xml") ? generate(Paths.get(bomInput)) : generate(bomInput);
```

Comment:
We should probably start using command line flags for this.

Reply:
Added command line parsing class `DashboardArguments`